test/libsolidity/smtCheckerTests/types/struct/array_struct_array_struct_storage_safe.sol

```diff
@@ -52,5 +52,6 @@ contract C {
 }
 // ====
 // SMTEngine: all
+// SMTTargets: assert
```

Comment:
Similar to above, the goal here is to test the assertions, not the array accesses.